src/sentry/rules/conditions/event_frequency.py

```diff
@@ -57,44 +57,51 @@ def passes(self, event, state):
         if not interval:
             return False
 
-        current_value = self.get_rate(event, interval)
+        current_value = self.get_rate(
+            event,
+            interval,
+            self.rule.environment_id,
```

Comment:
`rule` is technically not required by the constructor in `RuleBase` but the only time it isn't provided is for testing `data` parameter validity in the API serializer. There's a lot of weird stuff going on with this abstraction but I don't know as it's worth redoing right now just to enforce that `rule` is a required argument.